### PR TITLE
asadiqbal08/Bootcamp footer basic layout + hubspot Form

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### What are the relevant tickets?
[Ticket-1](https://trello.com/c/vQ0dvZyP/95-site-footer-layout-structure)
[Ticket-2](https://trello.com/c/5cg89xMZ/83-add-hubspot-form-to-footer)

#### What's this PR do?
Added a basic footer layout

#### Screenshots (if appropriate)
Screen Shot 2020-05-21 at 4 14 52 PM

Screen Shot 2020-05-21 at 4 48 27 PM

screencapture-bc-odl-local-8099-bootcamps-bootcamp-1-2020-05-22-14_56_49

### CODE REVIEW

**Comment by asadiqbal08** [comment]
@pdpinch in the attached screenshot on this PR, I need few things for clarification. 
- I can see some links on left column `MIT Online Innovation` what will I expect here ?
- I believe, the text below `Help & FAQ` and `Contact Us` would be either something different or not required. 
Let me know about the layout design.

**Comment by asadiqbal08** [comment]
updated the screenshots as per new design layout.

**Comment by pdpinch** [comment]
Does this screenshot include the live form from Hubspot?

File: main/templates/footer.html [hunk 1]
```diff
@@ -1,28 +1,70 @@
...
-        <a href="http://www.mit.edu" target="_blank"><img src="{% static 'images/mit-logo-ltgray-white@72x38.svg' %}" alt="MIT" width="72" height="38"></a>
-        <div class="footer-links">
-          <a href="{% url 'bootcamp-tos' %}" target="_blank">Bootcamps Terms of Service</a>
-          <a href="https://www.edx.org/" target="_blank">edX</a>
-          <a href="https://odl.mit.edu/" target="_blank">Office of Digital Learning</a>
+    <div class="row footer-top">
+      <div class="col-sm-8">
+        <div class="row">
+          <div class="col-6 links-info">
+            <h2><a href="https://xpro.zendesk.com/hc/">Help & FAQ</a></h2>
```
Comment (pdpinch) [inline main/templates/footer.html:9]:
```suggestion
            <h2><a href="https://mitbootcamps.zendesk.com/hc/">Help & FAQ</a></h2>
```

File: main/templates/footer.html [hunk 2]
```diff
@@ -1,28 +1,70 @@
...
+          </div>
+          <div class="col-6 links-info">
+            <h2><a href="https://xpro.zendesk.com/hc/en-us/requests/new">Contact Us</a></h2>
```
Comment (pdpinch) [inline main/templates/footer.html:12]:
```suggestion
            <h2><a href="https://mitbootcamps.zendesk.com/hc/requests/new">Contact Us</a></h2>
```

File: main/templates/footer.html [hunk 3]
```diff
@@ -1,28 +1,70 @@
...
+                <li><a href="/privacy-policy">Privacy Policy</a></li>
+              </ul>
+            </div>
+          </div>
+        </div>
+        <div class="col-md-4">
+          <div class="address-area">
+            <h3>MIT Bootcamps</h3>
+            <address>
+                <b>Massachusettes Institute Of Technology </br>
```
Comment (pdpinch) [inline main/templates/footer.html:50]: In your screenshot, this text is wrapping when it doesn't need to. Can we let it extend? Not a big deal.

**Comment by pdpinch** [comment]
Mobile screenshots?

File: main/templates/footer.html [hunk 4]
```diff
@@ -1,28 +1,70 @@
...
+    </div>
+    <div class=" footer-bottom">
+      <div class="row">
+        <div class="col-sm-8">
+          <div class="row">
+            <div class="col-6">
+              <ul class="list-unstyled links">
+                <li><a href="/about-us">About MIT Bootcamps</a></li>
+                <li><a href="/admissions">Admissions</a></li>
+                <li><a href="#">Bootcamps Programs</a></li>
```
Comment (pdpinch) [inline main/templates/footer.html:35]:
```suggestion
                <li><a href="bootcamps-programs">Bootcamps Programs</a></li>
```

I don't know if we'll use this URL, but might as well try.

File: main/templates/footer.html [hunk 5]
```diff
@@ -1,28 +1,70 @@
...
+          </div>
+        </div>
+        <div class="col-md-4">
+          <div class="address-area">
+            <h3>MIT Bootcamps</h3>
+            <address>
+                <b>Massachusettes Institute Of Technology </br>
+                77 Massachusettes Ave Cambridge, MA 02139, USA</b>
+            </address>
+            <a class="mit-open-learning" href="#">MIT Open Learning</a>
```
Comment (pdpinch) [inline main/templates/footer.html:53]:
```suggestion
            <a class="mit-open-learning" href="https://openlearning.mit.edu">MIT Open Learning</a>
```

File: main/templates/footer.html [hunk 6]
```diff
@@ -1,28 +1,70 @@
...
+          </div>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col-12">
+          <ul class="list-unstyled social-networks">
+            <li class="facebook"><a href="https://facebook.com/MITBootcamps"><i class="fa fa-facebook-f"></i></a></li>
+            <li class="linkedin"><a href="https://linkedin.com/company/11182752"><i class="fa fa-linkedin"></i></a></li>
+            <li class="twitter"><a href="https://twitter.com/MITBootcamps"><i class="fa fa-twitter"></i></a></li>
+            <li class="youtube"><a href="#"><i class="fa fa-youtube"></i></a></li>
```
Comment (pdpinch) [inline main/templates/footer.html:63]:
```suggestion
            <li class="youtube"><a href="https://www.youtube.com/MITBootcamps"><i class="fa fa-youtube"></i></a></li>
```

File: main/templates/footer.html [hunk 7]
```diff
@@ -1,28 +1,70 @@
...
+            <li class="instagram"><a href="https://instagram.com/MITBootcamps"><i class="fa fa-instagram"></i></a></li>
+            <li class="soundcloud"><a href="#"><i class="fa fa-soundcloud"></i></a></li>
```
Comment (pdpinch) [inline main/templates/footer.html:65]:
```suggestion
            <li class="soundcloud"><a href="https://soundcloud.com/MITBootcamps"><i class="fa fa-soundcloud"></i></a></li>
```

File: main/templates/footer.html (hunk 3 above)
Comment (asadiqbal08) [reply main/templates/footer.html:50]: I can extend but in the design it is showing me as per screenshot.

**Comment by asadiqbal08** [comment]
> Mobile screenshot

added

> Does this screenshot include the live form from Hubspot?

yes

**Comment by asadiqbal08** [comment]
Should we merge it and update the footer in later stages of development (incremental) ?

File: cms/templates/product_page.html [hunk 8]
```diff
@@ -31,3 +31,6 @@ <h3>{{ page.bootcamp_run.start_date|date }} - {{ page.bootcamp_run.end_date|date
   {% if three_column_image_text_section %} {% include "partials/three-column-image-text-section.html" with page=three_column_image_text_section %} {% endif %}
   {% if page.instructors %} {% include "partials/instructor.html" with page=page.instructors %} {% endif %}
 {% endblock %}
+{% block footer %}
+    {% include "footer.html" %}
+{% endblock %}
```
Comment (umarmughal824) [inline cms/templates/product_page.html:36]: NIT: empty line at end of file

**Comment by asadiqbal08** [comment]
Thanks @umarmughal824 

> Contact Us section is not at the exact center position

regarding this, I am expecting some changes that can be done further. 

> Mailing List is missing

It is here, you need to configure the hubspot to render the form. 

> Links below FAQ and Contact Us section is missing

I am sure, we need to get rid of these links.

**Comment by umarmughal824** [comment]
@pdpinch Links below FAQ and Contact Us section are missing, could you please confirm are these no more needed?

**Comment by asadiqbal08** [comment]
Actually, I was not sure either but after seeing @pdpinch comment over the design then I assume we do not need these links below. 

Screen Shot 2020-05-28 at 5 29 56 PM

**Comment by pdpinch** [comment]
The hubspot form looks odd to me. Is that the way it looks coming from the hubspot site?

I'll confirm that we're removing the text "Learn More Here" and "Email Bootcamps". It's possible some other text will be put in there later, but I don't want to hold this up any longer waiting.

**Comment by asadiqbal08** [comment]
I had a discussion @umarmughal824 and agree upon the current position of `Contact Us` section as per bootstrap standards. 
@pdpinch Do you have any concerns over the footer then let me know else I am going to merge this PR.

**Pull request merged**